blueoil/turbo_color_map.py

```diff
@@ -0,0 +1,258 @@
+TURBO_CMAP_DATA = [
```

Comment:
Maybe we need to set symlink to here
https://github.com/blue-oil/blueoil/tree/master/output_template/python/blueoil

Reply:
Ah, I get it. That will make sure the tests will pass (the ones that have failed, because the symlinks are missing there).

Thank you!!